gravityflow/gravityflow
# Fix HTML Fields on Workflow Detail Page

## Description
Re: [Issue#85](https://github.com/gravityflow/backlog/issues/85)
HTML fields with conditional logic display regardless of whether the condition is true or false on Workflow detail.

## Testing instructions
1. Create a form with a text field and say two HTML fields A and B.
2. Set field A to be displayed only if the text field is empty.
3. Set field B to be displayed only if the text field is not empty.
4. Add an Approval Workflow Step.
5. Process the form and the workflow step.
6. At the end of the workflow, we will be seeing both the HTML fields (instead of only one).
7. Update to this branch, and now we will only see HTML field A when the text field is empty. When the text field is not empty, we will only see HTML field B.

## Automated Test Enhancements
Added test `0053ConditionalHTMLWorkflowDetailCept.php` to check for conditional HTML display on Workflow Details page.
 
## Screenshots
N/A

## Documentation Changes
N/A

## Checklist:
